transactions/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+# Create your models here.
+from blocks.models import Block
+
+
+class Transaction(models.Model):
+    '''
+    This represents a transaction(exchange of coins) on the blockchain
+    '''
+    hash = models.CharField(max_length=100, blank=True, null=True)
+    block = models.ForeignKey(Block, blank=True, null=True, on_delete=models.CASCADE, related_name='transactions')
+    tx_ins_ct = models.IntegerField(blank=True, default=0)
```

Comment:
Do add a comment for any non-contextual engineer on what these two fields are.

Reply:
Also, why they are there.

Reply:
done